ctti/type_id.hpp

```diff
@@ -49,10 +50,9 @@ namespace ctti
         template<typename T>
         constexpr ctti::type_id_t type_id()
         {
-            const auto str = detail::make_string(CTTI_PRETTY_FUNCTION);
-            const auto name = str.template substr<CTTI_PRETTY_FUNCTION_BEGIN, CTTI_PRETTY_FUNCTION_END>();
-
-            return {name};
+            // one-liner required by MSVC :(
```

Comment:
I prefer a one liner. It was written in that way since GCC throwed ICEs with `make_string().substr()`.